gemrb/GUIScripts/bg2/Start.py

```diff
@@ -21,32 +21,31 @@
 import GameCheck
 from GUIDefines import SV_SAVEPATH
 
-skip_videos = False
+skipVideos = False
+skipIntro = 0
```

Comment:
This is for the game-specific intro, but now we have two very similar vars and approached. I proposed you either also use skipIntro in OnLoad or use SeenIntroVideos in RunStart2. Either can be treated as a bitfield and both persist.

Reply:
I now use `SeenIntroVideos` for the game specific ones, too.